src/components/checkout/CheckoutBlock.tsx

```diff
@@ -244,6 +244,11 @@ const CheckoutBlock: React.VFC<{
       ReactPixel.track('AddPaymentInfo', {
         value: check ? sum(check.orderProducts.map(prop('price'))) - sum(check.orderDiscounts.map(prop('price'))) : 0,
         currency: 'TWD',
+        contents: check.orderProducts.map(orderProduct => ({
+          id: orderProduct.productId,
+          name: orderProduct.name,
+          quantity: 1,
```

Comment:
should get product quantity from property `options` 
```ts
quantity: orderProduct?.options?.quantity || 1,
```

Reply:
`Number(orderProduct?.options?.quantity?.toString())`

ex. { quantity: "123"}